contentcuration/contentcuration/frontend/channelList/router.js

```diff
@@ -21,7 +21,7 @@ const router = new VueRouter({
     {
       name: RouteNames.CHANNEL_SETS,
       path: '/collections',
-      component: ChannelSetList,
```

Comment:
I assume that `ChannelSetList` and `ChannelSetItem` files (+ related obsolete test suites) can be removed since they won't be used anymore from anywhere - can you check on it and delete if that is so?